python/tvm/auto_scheduler/measure.py

```diff
@@ -652,7 +652,6 @@ def local_build_worker(args):
         The build result of this Builder thread.
     """
     inp, build_func, timeout, verbose = args
-    assert any(build_func == name for name in BuildFunc.name)
```

Comment:
it is removed by `black` tool automatically?

Reply:
any is error, so I change it